dll/win32/kernel32/client/except.c

```diff
@@ -697,14 +697,16 @@ UnhandledExceptionFilter(IN PEXCEPTION_POINTERS ExceptionInfo)
  */
 VOID
 WINAPI
-RaiseException(IN DWORD dwExceptionCode,
-               IN DWORD dwExceptionFlags,
-               IN DWORD nNumberOfArguments,
-               IN CONST ULONG_PTR *lpArguments OPTIONAL)
+RaiseException(
+    _In_ DWORD dwExceptionCode,
+    _In_ DWORD dwExceptionFlags,
+    _In_ DWORD nNumberOfArguments,
+    _In_opt_ const ULONG_PTR *lpArguments)
 {
     EXCEPTION_RECORD ExceptionRecord;
 
     /* Setup the exception record */
+    RtlZeroMemory(&ExceptionRecord, sizeof(ExceptionRecord));
```

Comment:
Then, `= NULL` and `= 0` lines below could be removed.
Or, you could zero `ExceptionRecord.ExceptionInformation` only.
Is this zeroing even needed at all?

Reply:
Maybe at least comment the now duplicated assignments then, that Serge pointed to? (my gutfeeling tells me that you do hesitate to delete them entirely)